pkg/rpc/server/server.go

```diff
@@ -837,11 +837,11 @@ func (s *Server) invokeFunction(reqParams request.Params) (interface{}, *respons
 	tx := &transaction.Transaction{}
 	checkWitnessHashesIndex := len(reqParams)
 	if checkWitnessHashesIndex > 3 {
-		cosigners, err := reqParams[3].GetCosigners()
+		signers, err := reqParams[3].GetSigners()
 		if err != nil {
 			return nil, response.ErrInvalidParams
 		}
-		tx.Cosigners = cosigners
+		tx.Signers = signers
```

Comment:
I think we have to initialize `tx.Signers` with some stub sender anyway, otherwise some code working with senders can fail.